plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/CASTAIPReader.java

```diff
@@ -99,47 +98,11 @@ private TestCaseResult parseCASTAIPIssue(Node flaw) throws Exception {
         return null;
     }
 
-    private int cweLookup(String name) {
+    private CweNumber cweLookup(String name) {
         if (name == null || name.isEmpty()) {
-            return 0000;
+            return CweNumber.DONTCARE;
         }
-        switch (name.trim()) {
-            case "614":
-                return CweNumber.INSECURE_COOKIE;
-            case "78":
-                return CweNumber.COMMAND_INJECTION;
-            case "79":
-                return CweNumber.XSS;
-            case "89":
-                return CweNumber.SQL_INJECTION;
-            case "90":
-                return CweNumber.LDAP_INJECTION;
-                //        case "header-injection"          :  return 113;  // header injection
-                //        case "hql-injection"             :  return 0000; // hql injection
-                //        case "unsafe-readline"           :  return 0000; // unsafe readline
-                //        case "reflection-injection"      :  return 0000; // reflection injection
-                //        case "reflected-xss"             :  return 79;   // xss
-            case "91":
-            case "643":
-                return CweNumber.XPATH_INJECTION;
-            case "73": // This tool calls this CWE-73 "External Control of File"
-            case "22":
-                return CweNumber.PATH_TRAVERSAL;
-                // Name or Path"
-                //        case "crypto-bad-mac"            :  return 328;  // weak hash
-                //        case "crypto-weak-randomness"    :  return 330;  // weak random
-                //        case "crypto-bad-ciphers"        :  return 327;  // weak encryption
-            case "501":
-                return CweNumber.TRUST_BOUNDARY_VIOLATION;
-                //        case "xxe"                       :  return 611;  // xml entity
-            case "134":
-                return CweNumber
-                        .EXTERNALLY_CONTROLLED_STRING; // Use of Externally-Controlled Format String
-                // - Which really isn't a
-            default:
-                System.out.println(
-                        "No matching CWE # found in CAST AIP Reader for: 'CWE-" + name + "'");
-        }
-        return 0000;
+
+        return CweNumber.lookup(name.trim());
```

Comment:
Again, I think its better to leave the manual mapping here so we can map in anything 'new'.

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379